.github/workflows/windows.yml

```diff
@@ -15,10 +15,9 @@ jobs:
         run: choco install 7zip wget python
       - name: Get snap7
         run: |
-          wget.exe --content-disposition -c https://sourceforge.net/projects/snap7/files/1.4.2/snap7-full-1.4.2.7z/download
+          wget.exe -O snap7-full-1.4.2.7z -c https://sourceforge.net/projects/snap7/files/1.4.2/snap7-full-1.4.2.7z/download
```

Comment:
if I look at the latest master build https://github.com/gijzelaerr/python-snap7/runs/1340797153?check_suite_focus=true

the file is saved with the right name. Are you sure you are using `wget.exe` and not the powershell alias? also the extraction works fine:

```
2020-11-02 07:41:02 (4.99 MB/s) - 'snap7-full-1.4.2.7z' saved [20591252/20591252]


7-Zip 19.00 (x64) : Copyright (c) 1999-2018 Igor Pavlov : 2019-02-21

Scanning the drive for archives:
1 file, 20591252 bytes (20 MiB)

Extracting archive: snap7-full-1.4.2.7z
```

Reply:
I noticed an error at this build: https://github.com/gijzelaerr/python-snap7/runs/1457872062?check_suite_focus=true . And I checked it on another computer. It seems to me Sourceforge servers started sending an invalid file name.
![wgetwtf](https://user-images.githubusercontent.com/52915342/100590362-f6e0b100-32fc-11eb-9ef2-58acd60ac5e5.PNG)

Reply:
ok then it makes sense. note that I added  `--content-disposition `  to make sure wget follows the redirects, but the documentation is a bit minimal on this.

Reply:
You're right, better keep it. I've checked we can have both options.